### ISSUE TITLE
Contact us. Error message do not provide ideas to recover from input errors

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#93]
### Description of issue

Error messages are not very descriptive or unhelpful to guide users to recover from errors.

![image](https://user-images.githubusercontent.com/5230720/137021193-c667914a-7a9c-4983-aa11-e4c9a81cc0f0.png)
![image](https://user-images.githubusercontent.com/5230720/137021204-19c994ec-caf1-4153-a5a0-30d646d7c276.png)
![image](https://user-images.githubusercontent.com/5230720/137021221-87237bbf-12fc-495b-a47e-70b861de29e1.png)

"This cannot be empty" messages could help the user understand what data we want from them:

Some ideas: 
- Your name: "Enter your name"
- Your email: "Enter your email address"
- Your message: "Enter what we can help you with" or "Enter the question you have for the Notify team"

Are more descriptive, even if they mostly repeat the label of the field.

### Potential fix

1. Add better error messages
2. Add `aria-required="true"` attribute to all fields, in addition to the `required` attribute. 

### Resources
- https://design-system.service.gov.uk/components/error-message/
- https://axe.deque.com/rule-help/en/aria-required-missing

**Comment by amazingphilippe** [comment]
@YedidaZalik What can we do for a radio selection like this?

<img width="544" alt="Capture d’écran, le 2021-11-09 à 08 48 02" src="https://user-images.githubusercontent.com/5230720/140935550-45069510-5677-48cb-add7-23ae40cbf65a.png">

**Comment by EliseKa** [comment]
Just to make sure - here there can only be 1 option selected, right?

**Comment by EliseKa** [comment]
I think "Sélectionnez une option" / "Choose an option" should be clear enough. I'll leave it to @YedidaZalik to decide if "Select" or "Choose" is better :)

**Comment by YedidaZalik** [comment]
I agree with "Choose an option".

if we're worried that they might pick more than one, but they're only allowed to pick one, then would it help to replace "an" and "une" with "1" ?

**Comment by amazingphilippe** [comment]
Should we spell it or use the numeral?
